.github/workflows/cmd-action.yml

```diff
@@ -1,15 +1,45 @@
 name: Command-Action
 
-on: 
+on:
   pull_request:
+  issue_comment:
+    types: [created]
 
 jobs:
   cmd-check:
     runs-on: ubuntu-latest
+    outputs:
+      commands: ${{ steps.cmd.outputs.commands }}
+      branch: ${{ steps.branch.outputs.branch }}
     name: Bot
     steps:
-      - uses: actions/checkout@v4
+      - uses: actions/checkout@v4.1.1
+      # Switch to the PR branch if it's triggered by a comment
+      - run: gh pr checkout ${{ github.event.issue.number || github.event.pull_request.number }}
+        if: ${{ github.event.issue.pull_request }}
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
       - uses: paritytech/cmd-action@main
+        id: cmd
         with: 
           commands-directory: '.github/scripts'
           GITHUB_TOKEN: ${{ github.token }}
+      - name: Export branch name
+        run: echo "branch=$(git rev-parse --abbrev-ref HEAD)" >> "$GITHUB_OUTPUT"
```

Comment:
Why do we need to parse branch here? Don't we have it in the variables?

Reply:
No, when it‘s generated from an issue comment we only have the PR number, but not the branch name, so I need to obtain it somehow. I could also do `gh pr view —json headRefName —jq „.headRefName“` to obtain the same result.